terraform/cellxgene.tf.json.template.py

```diff
@@ -150,13 +150,38 @@ def matrix_files():
                 "private_zone": False
             }
         },
-        "aws_iam_role": {
-            "cellxgene": {
-                "name": "ecsTaskExecutionRole"
+        "aws_iam_policy_document": {
+            "ecs-task-assume-role": {
+                "statement": {
+                    "actions": ["sts:AssumeRole"],
+                    "principals": {
+                        "type": "Service",
+                        "identifiers": [
+                            "ecs-tasks.amazonaws.com"
+                        ]
+                    }
+                }
+            }
+        },
+        "aws_iam_policy": {
+            "ecs-task-execution-role-policy": {
+                "arn": "arn:aws:iam::aws:policy/service-role/AmazonECSTaskExecutionRolePolicy"
```

Comment:
It does not make sense to look up an `aws_iam_policy` data source by ARN if that ARN is well-known and documented and …